sources/install.sh

```diff
@@ -418,8 +418,8 @@ function install_bolt() {
 
 function install_crackmapexec() {
   colorecho "Installing CrackMapExec"
-  python3 -m pipx install crackmapexec
-  ~/.local/bin/crackmapexec
+  git -C /opt/tools/ clone https://github.com/Porchetta-Industries/CrackMapExec.git
+  python3 -m pipx install git+https://github.com/Porchetta-Industries/CrackMapExec.git
```

Comment:
Let's have pipx install from sources local sources, so that users have the ability to edit sources and recompile
--> `python3 -m pipx install /opt/tools/CrackMapExec/`